src/std/text.ab

```diff
@@ -94,6 +94,7 @@ pub fun chars(text: Text): [Text] {
 /// Gets the length of provided text or array.
 #[allow_absurd_cast]
 pub fun len(value): Num {
+    echo "The len stdlib is deprecated, use the builtin!"
```

Comment:
We're still in alpha; surely it would be better to remove the standard library function completely, rather than deprecating it?

Reply:
We don't have anything to deprecate stuff right now in Amber so I thought that an alert it was the only option

Reply:
+1 on this, there is no need to deprecate. we are way too early for that

Reply:
Yep. I agree with @hdwalters and @b1ek